modules/transport-netty4/src/test/java/org/elasticsearch/transport/netty4/Netty4SizeHeaderFrameDecoderTests.java

```diff
@@ -19,6 +19,7 @@
 
 package org.elasticsearch.transport.netty4;
 
+import org.elasticsearch.MockSocket;
```

Comment:
Split packages are packages that live in two jars. Yes, it needs to be fixed in mocksocket but this seemed a good place to raise the conversation.